### ISSUE TITLE
Check that a 1.3 app works on Galaxy

### ISSUE THREAD

**Issue by zol** [meteor/meteor#6299]
It should exercise ES2015 syntax, imports, exports and the new Cordova system.

**Comment by devwax** [comment]
Hi. Rohit pointed me here. I was the one that asked about 1.3 beta on Galaxy when my app crashed on deployment the other day. Personally, this is low/no-priority as a traditional support issue as...

- the site is just a personal blog used for experimenting
- I assumed that 1.3 wouldn't work yet anyway, as it's still in bleeding-edge beta
- and I can just deploy in 1.2, no problem.

But if I can be of assistance in preparing 1.3 for Galaxy I'm more than happy to help.

Deployment Details:
- Meteor Version: 1.3-beta.11
- Mongo: MongoLab.com sandbox database

- Deployment Settings: *(details censored, obv)*
```console
{
  "galaxy.meteor.com": {
    "env": {
      "ROOT_URL": "http://[sitename].com",
      "MONGO_URL": "mongodb://<username>:<dbpassword>@********.mongolab.com:******/dbname"
    }
  }
}
```

Deployment command:
```console
DEPLOY_HOSTNAME=galaxy.meteor.com meteor deploy [sitename].com --settings ../private/settings-prod.json
```

Error upon deployment in container logs:
```console
2016-02-21 16:22:17-08:00 
ark1 
2016-02-21 16:22:17-08:00/app/bundle/programs/server/boot.js:286 
ark1 
2016-02-21 16:22:17-08:00}).run(); 
ark1 
2016-02-21 16:22:17-08:00 ^ 
ark1 
2016-02-21 16:22:17-08:00Error: Cannot find module '/app/bundle/programs/server/npm/babel-compiler/node_modules/meteor-babel' 
ark1 
2016-02-21 16:22:17-08:00 at Function.Module._resolveFilename (module.js:338:15) 
ark1 
2016-02-21 16:22:17-08:00 at Function.Module._load (module.js:280:25) 
ark1 
2016-02-21 16:22:17-08:00 at Module.require (module.js:364:17) 
ark1 
2016-02-21 16:22:17-08:00 at require (module.js:380:17) 
ark1 
2016-02-21 16:22:17-08:00 at Object.Npm.require (/app/bundle/programs/server/boot.js:165:18) 
ark1 
2016-02-21 16:22:17-08:00 at BabelCompiler.extraFeatures (packages/babel-compiler/babel.js:1:1) 
ark1 
2016-02-21 16:22:17-08:00 at packages/babel-compiler/babel.js:44:1 
ark1 
2016-02-21 16:22:17-08:00 at packages/babel-compiler/babel-compiler.js:98:1 
ark1 
2016-02-21 16:22:17-08:00 at /app/bundle/programs/server/boot.js:245:10 
ark1 
2016-02-21 16:22:17-08:00 at Array.forEach (native) 
ark1 
2016-02-21 16:22:17-08:00Application exited with code: 8 signal: null 
ark1 
2016-02-21 16:22:17-08:00Application process closed with code: 8 signal: null 
ark1 
2016-02-21 16:22:21-08:00The container has crashed. A new container will be started to replace it.
```

Packages
```console
meteor-base             # Packages every Meteor app needs to have
mobile-experience       # Packages for a great mobile UX
mongo                   # The database Meteor supports right now
blaze-html-templates    # Compile .html files into Meteor Blaze views
session                 # Client-side reactive dictionary for your app
jquery                  # Helpful client-side library
tracker                 # Meteor's client-side reactive programming library

standard-minifier-css   # CSS minifier run for production mode
standard-minifier-js    # JS minifier run for production mode
es5-shim                # ECMAScript 5 compatibility for older browsers.
ecmascript              # Enable ECMAScript2015+ syntax in app code

#autopublish             # Publish all data to the clients (for prototyping)
#insecure                # Allow all DB writes from clients (for prototyping)

#accounts-password
#jparker:gravatar
#react-meteor-data

less
kadira:flow-router
kadira:dochead
```

NPM Modules
```console
  "dependencies": {
    "algoliasearch": "^3.12.0",
    "algoliasearch-helper": "^2.8.1",
    "react": "^0.14.7",
    "react-addons-update": "^0.14.7",
    "react-dom": "^0.14.7",
    "react-mixin": "^2.0.2",
    "react-mounter": "^1.0.0"
  },
  "devDependencies": {},
```

If you have any questions or if any further details would help let me know.

**Comment by zol** [comment]
Thanks for adding some more detail!

**Comment by devwax** [comment]
As mentioned in #6327 when I remove both standard-minifier packages, the deployment is successful.

```console
meteor-base             # Packages every Meteor app needs to have
mobile-experience       # Packages for a great mobile UX
mongo                   # The database Meteor supports right now
blaze-html-templates    # Compile .html files into Meteor Blaze views
session                 # Client-side reactive dictionary for your app
jquery                  # Helpful client-side library
tracker                 # Meteor's client-side reactive programming library

# Removed to see if they were breaking deployment on galaxy (they were)
#standard-minifier-css   # CSS minifier run for production mode
#standard-minifier-js    # JS minifier run for production mode

es5-shim                # ECMAScript 5 compatibility for older browsers.
ecmascript              # Enable ECMAScript2015+ syntax in app code

#autopublish             # Publish all data to the clients (for prototyping)
#insecure                # Allow all DB writes from clients (for prototyping)

#accounts-password
#jparker:gravatar
#react-meteor-data

less
kadira:flow-router
kadira:dochead
```

**Comment by devwax** [comment]
**Update:** at 4:00pm PST I tried adding both minifiers back again and the deployment works now with 1.3-beta.11 on Galaxy. I will report back if I encounter any further issues. Resolves #6299.

**Comment by zol** [comment]
Closing this ticket as I found some issues as described in #6380
